spark/src/main/scala/io/delta/tables/execution/VacuumTableCommand.scala

```diff
@@ -36,6 +37,8 @@ import org.apache.spark.sql.types.StringType
 case class VacuumTableCommand(
     override val child: LogicalPlan,
     horizonHours: Option[Double],
+    reservoirTable: Option[LogicalPlan],
```

Comment:
The external API is called "inventory" but internally it's called "reservoir". IIUIC the term "reservoir" is used in this code but it's meant to be the set of files that are _actually_ in the reservoir (e.g. `reservoirBase` is the base path of the reservoir). Would it make sense to call this "inventory", to match the syntax? And then the "reservoir files" are given by the intersection of the inventory with the reservoirBase.

Reply:
@bart-samwel :) I was a bit skeptical about using inventory as it was too AWS-specific (rightly pointed out by you in the other discussion thread). So reservoir was the other name that popped into my mind. If you are OK with the reservoir I will update the grammar to use the reservoir instead of the AWS specific inventory term. If you think we can go with Inventory ;) then I will update the code and functions.